lib/pgreset.rb

```diff
@@ -8,7 +8,7 @@ def drop
 
         database_name =
           begin
-            self.configuration_hash[:database] || self.configuration_hash.fetch('database')
+            configuration_hash[:database] || configuration_hash.fetch('database')
```

Comment:
I tried this, it was causing another trouble for rails < 6.1. Because it will throw "Undefined local variable" instead of "NoMethodError" specified here.

FYI: https://github.com/dafalcon/pgreset/pull/3